src/Sylius/Bundle/ApiBundle/Swagger/AcceptLanguageHeaderDocumentationNormalizer.php

```diff
@@ -51,7 +56,7 @@ public function normalize($object, $format = null, array $context = [])
 
         foreach ($docs['paths'] as $path => $methods) {
             foreach ($methods as $methodName => $methodBody) {
-                if (is_object($methodBody)) {
+                if (is_object($methodBody) && method_exists($methodBody, 'getArrayCopy')) {
```

Comment:
Extra `if`'s operator might change logic of the function.

```php
if (is_object($methodBody)) {
Assert::methodExists($methodBody, 'getArrayCopy']
```

WDYT?

Reply:
Right!